persistence/src/main/scala/hmda/persistence/institutions/SubmissionPersistence.scala

```diff
@@ -54,7 +54,7 @@ class SubmissionPersistence(institutionId: String, period: String) extends HmdaP
     case CreateSubmission =>
       val seqNr = state.submissions.size + 1
       val submissionId = SubmissionId(institutionId, period, seqNr)
-      val newSubmission = Submission(submissionId, Created)
+      val newSubmission = Submission(submissionId, Created, 0L, 0L)
```

Comment:
I believe that this line assigns all `Submission`s a creation time of `0L`. Is that correct?